src/main/java/org/folio/innreach/specification/InnReachTransactionSpecification.java

```diff
@@ -37,20 +37,20 @@
   static Specification<InnReachTransaction> fieldsLookup(InnReachTransactionFilterParameters parameters) {
     return (transaction, cq, cb) -> {
       var hold = transaction.join("hold");
-      var itemHold = cb.treat(hold, TransactionItemHold.class);
-      var localHold = cb.treat(hold, TransactionLocalHold.class);
+      var transactionHold = cb.treat(hold, TransactionHold.class);
```

Comment:
remove this variable, use `hold`

Reply:
hold already exist

Reply:
yes it exist and it should be possible to use it. This `treat` make no sense now as it's needed only for downcasting